package.json

```diff
@@ -110,7 +110,6 @@
     "@graphql-codegen/typescript-operations": "^3.0.2",
     "@types/cors": "^2.8.13",
     "@types/gl": "^6.0.2",
-    "@types/jpeg-js": "^0.3.7",
```

Comment:
Would this break the build if the optional dependencies are not installed? (I think it would)

Reply:
also there is a looming issue that the packages might provide their own typing and @types would not be available/outdated

Reply:
I apologize. It will not work.
Previously, `npm run build` was telling me this: 
`npm WARN deprecated @types/jpeg-js@0.3.7: This is a stub types definition. jpeg-js provides its own type definitions, so you do not need this installed.` 
So I tried removing `@types/jpeg-js`, running `npm install --omit=optional`, and building and it worked. But perhaps I just forgot to delete something. Now I tried again and it doesn't work.
I rolled this change back.

Reply:
You prolly had the older version of `node_modules`